modelling/src/main/java/org/axonframework/modelling/entity/SimpleEntityModel.java

```diff
@@ -135,11 +173,31 @@ public MessageStream.Single<CommandResultMessage<?>> handle(@Nonnull CommandMess
         return MessageStream.failed(new NoHandlerForCommandException(message, entityType));
     }
 
+    @Nullable
+    @Override
+    public E evolve(@Nonnull E entity, @Nonnull EventMessage<?> event, @Nonnull ProcessingContext context) {
```

Comment:
Can you help me understand why we need to override the `evolve` method here since these changes?